### DISCUSSION TITLE
Need help with conditional `autoClosingPairs` in language configuration json

### DISCUSSION THREAD

**Discussion by ttax00** [microsoft/vscode-discussions#484] [Extension Development QnA]
> Issue link: https://github.com/TechTheAwesome/code-yew-server/issues/19

Hi everyone, I'm working on a language feature support for yew's `html!` rust, and ran into an issue with closing pairs language configuration. Since the empty fragment tag `<></>` should only be auto closed within the `html! { }` block, and not outside. So something like after `html! {` and before `}` where fragment tag will be evaluated would be helpful. 

As far as I can tell, in the past few days of digging, there's only 2 option for `notIn` ("string" & "comment"). And an alternative could be implement a format on type function?  But my intuition tells me the api is not being used as intended in that case.

Many thanks ahead of time!!

**Comment by isidorn** [comment]
@hediet might be able to help here

**Comment by olguzzar** [comment]
Hi @ttax00, it's been a while since there's been any activity on this. Is this still an issue? We will close this out if there aren't any updates - thanks!

**Comment by burkeholland** [comment]
Closing as inactive. If you feel this is an error, please re-open.
